### PR TITLE
Crafted items now craft into your hands

### PR DESCRIPTION
**Pull request by Iamgoofball**
If your hands aren't free/you cant hold that item it drops on the floor

### CODE REVIEW

File: code/modules/crafting/craft.dm [hunk 1]
```diff
@@ -106,6 +106,7 @@
 				var/list/parts = del_reqs(R, user)
 				var/atom/movable/I = new R.result (get_turf(user.loc))
 				I.CheckParts(parts, R)
+				user.put_in_hands(I)
```
Comment (ChangelingRain) [inline code/modules/crafting/craft.dm]: make sure it's actually an item before you put it in hands, or you'll get runtimes

Comment (Iamgoofball) [inline code/modules/crafting/craft.dm]: if I read the put_in_hands() code it already does this but I'll doublecheck

**Comment by PKPenguin321** [comment]
i hope this accounts for items like skateboards that aren't intended to go into the hand

**Comment by PKPenguin321** [comment]
wait no yeah i actually looked at the code this seems fine

**Comment by Cheridan** [comment]
I think there's a put_in_hands_if_possible() proc you can use to get rid of the istype check

**Comment by KorPhaeron** [comment]
@iamgoofball

**Comment by Iamgoofball** [comment]
how the FUCK DID THIS CONFLICT WHAT

**Comment by PKPenguin321** [comment]
That's a good question because according to the file history the last time it was changed was 4 days ago. https://github.com/tgstation/tgstation/commit/8bcb4789bd00e18f72f5943b5b88ca97d20f006f

**Comment by PKPenguin321** [comment]
Reee

**Pull request closed without merging**